Patches/Vanilla Factions Expanded - Medieval/PawnKindDefs/Patch_Musketman.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+					<mods>
+						<li>Vanilla Factions Expanded - Medieval</li>
+					</mods>
+		<match Class="PatchOperationFindMod">
+					<mods>
+						<li>Vanilla Weapons Expanded</li>
+					</mods>
+					
+			<match Class="PatchOperationSequence">
+				<operations>
+              
+	<!-- ========== Give ammo to musketeer ========== -->
+        <li Class="PatchOperationAddModExtension">
+          <xpath>/Defs/PawnKindDef[defName="VFEM_Medieval_Musketeer"]</xpath>
+          <value>
+            <li Class="CombatExtended.LoadoutPropertiesExtension">
+              <primaryMagazineCount>
+                <min>8</min>
```

Comment:
Could probably use a bit more ammo, since each magazine is only going to be 1 shot each.

Reply:
Pointless to give them more considering how obscenely long the reload time is, musketman of the era don't carry much ammo anyway and are expected to fight in melee to some extent.